deploy/bicep/groups/networking.bicep

```diff
@@ -8,6 +8,8 @@ param projectName string
 param location string
 param virtualNetworkName string
 
+param cloudEndpoints object = loadJsonContent('../../cloud_endpoints.json')
```

Comment:
I think for groups/modules, it might be cleaner to pass the endpoints object in as a parameter vs having the tight coupling on the local file. Thoughts?

Reply:
I thought of passing the endpoint as a parameter using the [environment](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-functions-deployment#environment) function, but the PG suffix is not in the list. Hence, I decided to use a json file. Let me know if there is a better way to handle this?

Reply:
Yeah, it's too bad the environment function doesn't provide everything needed. I think the json file is fine for use in the entrypoint (main.bicep), but I would read it once there and then pass the object along to where it's needed in other groups or modules such as this one.

Reply:
yeah, that makes sense. I'll do the change.